lois.gemspec

```diff
@@ -1,4 +1,4 @@
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path('lib', __dir__)
```

Comment:
Is this another rubocop fix?

Reply:
That is correct, rubocop recommended this fix.